plugin/trino-hudi/src/test/java/io/trino/plugin/hudi/BaseHudiMinioConnectorTest.java

```diff
@@ -1,112 +0,0 @@
-/*
- * Licensed under the Apache License, Version 2.0 (the "License");
- * you may not use this file except in compliance with the License.
- * You may obtain a copy of the License at
- *
- *     http://www.apache.org/licenses/LICENSE-2.0
- *
- * Unless required by applicable law or agreed to in writing, software
- * distributed under the License is distributed on an "AS IS" BASIS,
- * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
- * See the License for the specific language governing permissions and
- * limitations under the License.
- */
-package io.trino.plugin.hudi;
-
-import com.google.common.collect.ImmutableMap;
-import io.trino.plugin.hive.containers.HiveMinioDataLake;
-import io.trino.plugin.hudi.testing.TpchHudiTablesInitializer;
-import io.trino.testing.BaseConnectorTest;
-import io.trino.testing.QueryRunner;
-import io.trino.testing.TestingConnectorBehavior;
-import org.apache.hudi.common.model.HoodieTableType;
-import org.testng.annotations.Test;
-
-import static io.trino.plugin.hudi.BaseHudiConnectorTest.columnsToHide;
-import static io.trino.plugin.hudi.S3HudiQueryRunner.HUDI_MINIO_TESTS;
-import static io.trino.testing.sql.TestTable.randomTableSuffix;
-import static java.util.Objects.requireNonNull;
-import static org.assertj.core.api.Assertions.assertThat;
-
-public abstract class BaseHudiMinioConnectorTest
-        extends BaseConnectorTest
```

Comment:
Can we extend `BaseHudiConnectorTest` instead of removing this class? We sometimes need to add tests requiring file systems (e.g. number of files in the specified directory) and this base class is helpful to provides such helper methods.

Reply:
I'll add it back and make it extend BaseHudiConnectorTest. Thanks for the context.